guide.md

```diff
@@ -0,0 +1,18 @@
+[Back](./README.md)
+
+# Guide
+
+1. The list is sorted with alphabetical sections. With exceptions for "Numbers", "Symbols" and "Others" which are special.
+2. In section, abbrs are sorted in alphabetical order.
+3. Abbrs are in word-abbr format.
```

Comment:
These should be obvious at first look and we don't need to state these directly.

Reply:
Only wanted to be clear but ok.